lib/registration.pm

```diff
@@ -114,6 +114,13 @@ sub fill_in_registration_data {
                 }
             }
             my @scc_addons = split(/,/, get_var('SCC_ADDONS', ''));
+
+            # Workaround for boo#1056047
```

Comment:
this comment is redundant because the `record_soft_failure` already tells it all